src/theme.js

```diff
@@ -88,6 +88,15 @@ export const gordonColors = {
   },
 };
 
+// Material-UI breakpoints
+export const windowBreakWidths = {
+  breakXS: 360,
+  breakSM: 600,
+  breakMD: 960,
+  breakLG: 1280,
+  breakXL: 1920,
+};
```

Comment:
Yes, that's exactly what I used them for since I can't import those variables from vars.scss to a JS file. I've updated the comment that was already there to reflect your suggestion :)